src/codeCache.h

```diff
@@ -24,13 +22,18 @@ enum ImportId {
     im_pthread_exit,
     im_pthread_setspecific,
     im_poll,
+    im_malloc,
```

Comment:
In case it is helpful, here are some of the functions we had to hook: https://github.com/DataDog/ddprof/blob/bccb05d89ec9e8473cd7f4f26d4e4948a70839c8/src/lib/symbol_overrides.cc#L852-L899
Though I realize you might want to have a full coverage for now.

Reply:
Thanks for the link. We could implement these in another iteration or prioritize based on customer requests.